cms/static/cms/js/require-config.js

```diff
@@ -241,14 +244,26 @@
                 exports: "XBlock",
                 deps: ["xblock/core"]
             },
-
             "coffee/src/main": {
                 deps: ["coffee/src/ajax_prefix"]
             },
             "js/src/logger": {
                 exports: "Logger",
                 deps: ["coffee/src/ajax_prefix"]
             },
+            "modernizr": {
+                exports: "Modernizr"
+            },
+            "afontgarde": {
+                exports: "AFontGarde"
+            },
+            "edxicons": {
+                exports: "edxicons"
+            },
+            "draggabilly": {
+                deps: ["jquery"],
+                exports: "Draggabilly"
```

Comment:
You should not need the exports here. Draggabilly is RequireJS aware, so if Require is present, it will not export anything.

The "exports" part is only for libraries that don't use RequireJS, and instead add their library on the global window object.